src/lazy/encoder/value_writer.rs

```diff
@@ -425,4 +425,28 @@ pub trait SequenceWriter: MakeValueWriter {
     ) -> IonResult<<Self::ValueWriter<'a> as ValueWriter>::EExpWriter> {
         self.value_writer().eexp_writer(macro_id)
     }
+
+    fn write_list<V: WriteAsIon, I: IntoIterator<Item = V>>(
+        &mut self,
+        values: I,
+    ) -> IonResult<&mut Self> {
+        self.value_writer().write_list(values)?;
+        Ok(self)
+    }
```

Comment:
🗺️ These methods were already available on `ValueWriter`, this made it possible to also call them directly on `SequenceWriter` impls.